CMake/kwiver-setup-tests-python.cmake

```diff
@@ -0,0 +1,29 @@
+#
+# Script to set up environment for python nosetests
+#
+
+
+# results of tests being run will be exported to an Xunit xml file
+if (NOSE_RUNNER)
+
+  set(no_install TRUE)
+  string(TOLOWER "${CMAKE_PROJECT_NAME}" project_name)
+
+  if (WIN32)
+    set(kwiver_test_output_path    "${KWIVER_BINARY_DIR}/$<CONFIG>/bin" )
```

Comment:
Does the inclusion of $<CONFIG> work here? It's a generator expression and not set at configure time. Usually it requires a custom target or command to get it right but I can't see clearly if that's how this stuff is getting called or not.

Reply:
I was basing this variable on the CMake [docs](https://cmake.org/cmake/help/v3.15/manual/cmake-generator-expressions.7.html#id2) and eventually it is passed to a call to either a custom command, or a call to add test/test_properties all of which I believe perform the expansion of the generator expressions. I may have, of course, misunderstood.

Reply:
That's fine ... that documentation is correct and $<CONFIG> is the right way to do what you are trying to do. As long as it's being used within a structure that runs during generation and is resolving correctly, all is well. I just couldn't easily follow the trail to verify that. Thanks!

Reply:
I thought about it a little more, and while, with the current usage of kwiver-setup-python-tests, that genex is fine, however as we transition to pytest, the usage of this setup file will change, and the genex will cause problems. I'll rework a more robust solution.